homeassistant/components/ecobee/number.py

```diff
@@ -0,0 +1,98 @@
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN, ECOBEE_MODEL_TO_NAME, MANUFACTURER
+
+_LOGGER = logging.getLogger(__name__)
+
+VENTILATOR_MIN_VALUE = 0
+VENTILATOR_MAX_VALUE = 60
+VENTILATOR_STEP = 5
+VENTILATOR_MEASUREMENT_UNIT = UnitOfTime.MINUTES
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the ecobee thermostat number entity."""
+    data = hass.data[DOMAIN]
+    entities = []
+    _LOGGER.debug("Adding min time ventilators numbers (if present)")
+    for index in range(len(data.ecobee.thermostats)):
+        thermostat = data.ecobee.get_thermostat(index)
+        if thermostat["settings"]["ventilatorType"] != "none":
+            _LOGGER.debug("Adding %s's ventilator min times number", thermostat["name"])
+            entities.append(
+                EcobeeVentilatorMinTime(
+                    data,
+                    index,
+                    "home",
+                    "ventilatorMinOnTimeHome",
+                    data.ecobee.set_ventilator_min_on_time_home,
+                )
+            )
+
+            entities.append(
+                EcobeeVentilatorMinTime(
+                    data,
+                    index,
+                    "away",
+                    "ventilatorMinOnTimeAway",
+                    data.ecobee.set_ventilator_min_on_time_away,
+                )
+            )
+
+    async_add_entities(entities, True)
+
+
+class EcobeeVentilatorMinTime(NumberEntity):
+    """A number class, representing min time  for an ecobee thermostat with ventilator attached."""
+
+    _attr_has_entity_name = True
+
+    def __init__(self, data, thermostat_index, mode, ecobee_setting_key, set_func):
+        """Initialize ecobee ventilator platform."""
+        self.data = data
+        self.thermostat_index = thermostat_index
+        self.thermostat = self.data.ecobee.get_thermostat(self.thermostat_index)
+        self.ecobee_setting_key = ecobee_setting_key
+        self.set_func = set_func
+        self._attr_name = f"Ventilator min time {mode}"
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, self.thermostat["identifier"])},
+            manufacturer=MANUFACTURER,
+            model=ECOBEE_MODEL_TO_NAME.get(self.thermostat["modelNumber"]),
+            name=self.thermostat["name"],
+        )
+        self._attr_unique_id = f'{self.thermostat["identifier"]}_{mode}'
+        self._attr_native_min_value = VENTILATOR_MIN_VALUE
+        self._attr_native_max_value = VENTILATOR_MAX_VALUE
+        self._attr_native_step = VENTILATOR_STEP
+        self._attr_native_value = self.thermostat["settings"][ecobee_setting_key]
+        self._attr_native_unit_of_measurement = VENTILATOR_MEASUREMENT_UNIT
```

Comment:
```suggestion
```

Reply:
Why did you remove the `native_unit_of_measurement`

Reply:
Was meant to go to class level as it's a fixed record.